client/app/components/login/login.html

```diff
@@ -3,24 +3,25 @@
   <div flex="nogrow" layout="column">
     <div flex id="loginContent" layout-padding class="md-whiteframe-2dp">
       <form login-form ng-submit="$ctrl.login()">
-        <h2>Login</h2>
+        <h2 translate>LOGIN.TITLE</h2>
```

Comment:
Ah forgot. Yes, that could be a good reason. I only thought of the case that a translation is broken and therefore the key is shown (which would still be better than nothing).